src/Card/Card.tsx

```diff
@@ -0,0 +1,192 @@
+/** @jsx jsx */
+import * as colors from "../colors";
+import { base } from "../typography";
+import { jsx } from "@emotion/core";
+import React from "react";
+import PropTypes from "prop-types";
+
+interface Props
+  extends React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  > {
+  /**
+   * The content of the card,
+   * appears below the title and description
+   */
+  children?: React.ReactNode;
+
+  heading?: React.ReactNode;
+
+  /**
+   * the description for this card
+   * appears in grey below the title
+   */
+  description?: React.ReactNode;
+
+  /**
+   * actions could be a button
+   * or a tooltip or anything the card should display
+   * aligned with the title on the right
+   */
+  actions?: React.ReactNode;
+
+  /**
+   * cards can be standard (18px text, larger padding)
```

Comment:
NIT: I'd leave out the 18px text here because then you have to remember to change it in two places. If you change it below and not here, you have outdated documentation, which is worse than no documentation. `size` can be self documenting too :) 

Push back when you think things don't make sense!